cloudhub-2/modules/ROOT/pages/ps-create-configure.adoc

```diff
@@ -35,6 +35,9 @@ include::partial$select-private-space.adoc[tag=clickPrivateSpaceName]
 --
 .. Select the xref:ps-gather-setup-info.adoc#private-network-region[region] where you want your apps to run.
 .. Enter the xref:ps-gather-setup-info.adoc#cidr-block[CIDR block], which is the range of IP addresses that your apps can use.
+.. Optionally, enter your corporate private network's reserved CIDRs to connect to your private space. 
```

Comment:
Optionally, reserve your corporate private network's CIDRs to connect to your private space.

Reply:
```suggestion
.. Optionally, reserve your corporate private network's CIDRs to connect to your private space by entering them in a comma-separated list. 
```

Reply:
can we also document the two approaches to specify reserved corporate CIDRs (from 1-pager)?

Reply:
I don't have a place to put those right now. That will take more explanation elsewhere. It cannot go in this list.